controller_manager/test/test_controllers_chaining_with_controller_manager.cpp

```diff
@@ -128,18 +128,18 @@ class TestControllerChainingWithControllerManager
 
     // configure Left Wheel controller
     controller_interface::InterfaceConfiguration pid_left_cmd_ifs_cfg = {
-      controller_interface::interface_configuration_type::INDIVIDUAL, {"wheel_left/velocity"}};
+      controller_interface::interface_configuration_type::INDIVIDUAL, {"wheel_0_joint/velocity"}};
```

Comment:
why these changes? there's references to wheel left and wheel right all around this code. Changing it to numbers only makes it harder to read IMO

Reply:
The joints are named differently in the URDF string in descriptions.hpp, URDF parser would throw an error here. Do you want me to revert these changes but fix the URDF?

Reply:
yeah, maybe better to fix urdf

Reply:
@bmagyar @destogl fixed